tools/RunTests.ps1

```diff
@@ -0,0 +1,194 @@
+<#
+.SYNOPSIS
+    Run tests from a specific .testdef file
+
+.DESCRIPTION
+    The RunTests script takes a testdef filename (without extension) and runs the tests defined in that
+    specific .testdef file. This is useful for development when you want to run tests for a specific
+    component rather than all tests.
+
+    The script searches for the specified .testdef file in the BuildOutput directory structure and
+    executes the tests according to the testdef configuration.
+
+.PARAMETER TestDef
+    The name of the .testdef file to run (without the .testdef extension)
+    Example: "Deployment" to run "Deployment.testdef"
+
+.PARAMETER OutputFolder
+    Set the base folder for the script to look for testdefs. Default: BuildOutput
+
+.PARAMETER Platform
+    Only run tests for the selected platform (x86, x64, arm64). Default: x64
+
+.PARAMETER Configuration
+    Only run tests for the selected configuration (Debug, Release). Default: Release
+
+.PARAMETER RunDisabled
+    If set, will run tests that are marked as "Disabled" in the testdef file. Default: false
+
+.EXAMPLE
+    .\RunTests.ps1 -TestDef "DeploymentTests"
+
+.EXAMPLE
+    .\RunTests.ps1 -TestDef "Deployment" -Platform "x86" -Configuration "Debug"
+#>
+
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$TestDef,
+
+    [Parameter(Mandatory=$false)]
+    [string]$OutputFolder = "BuildOutput",
+
+    [Parameter(Mandatory=$false)]
+    [string]$Platform = "x64",
+
+    [Parameter(Mandatory=$false)]
```

Comment:
Mandatory's default=false

RECOMMEND: Only specify for required parameters i.e. `[Parameter(Mandatory=$true)]`